.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.4.3-node-browsers
+    steps:
+      - checkout
+      - run:
+          # This is essential for capybara-webkit
+          name: Install System Dependencies
+          command: sudo apt-get install qt5-default libqt5webkit5-dev gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x
```

Comment:
I presume eventually you'll want to get these deps on the docker container directly so they don't need installed on each CI run?

Reply:
Yup, that would be the plan.